packs/hpe-icsp/actions/icsp_buildplan_apply.yaml

```diff
@@ -0,0 +1,28 @@
+---
+  name: "icsp_buildplan_apply"
+  runner_type: "run-python"
+  description: "Apply Buidlplan to Specified Servers"
+  enabled: true
+  entry_point: "icsp_buildplan_apply.py"
+  parameters:
+    buildplan_uris:
+      type: "array"
+      description: "Comma seperated list of build plan URIs"
```

Comment:
switched to id. Although not using the "items" feature yet as last time i used it there was a bug in the code where it couldn't map the elements the items type.